TACC/Core-Portal
# Quick: FP-1194: CMS Samples: Update Secrets, Add Settings

## Overview: ##

Update sample CMS secrets since https://github.com/TACC/Core-CMS/pull/341 refactored secrets.

## Related Jira tickets: ##

* [FP-1194](http://jira.tacc.utexas.edu/browse/FP-1194)

## Summary of Changes: ##

- Remove long-since-outdated and recently-outdated code from `secrets.sample.py`.
- Remove `_` prefix from `secrets.sample.py` variables.
- Add a `server/conf/cms/settings_custom.sample.py`.

## Testing Steps: ##
1. Load a Portal using the new CMS secrets.
2. Ensure CMS:
	- does not crash
	- provides all header content and styles
	- can be loaded and pages added and published

## UI Photos:

N/A

## Notes: ##

I have not tested this, because of limited bandwidth and there being no task for this. But @nathanfranklin needs its changes, so I'm being a butt and letting him test.
